base-docker-compose.yml

```diff
@@ -6,70 +6,34 @@ version: "3.4"
 #TODO: logging containers, consul...
 
 services:
-  #to get the password (hashed): (also remember to escape $ with anotehr $)
-  # docker run --rm httpd:2.4-alpine htpasswd -nbB admin 5ecr3t | cut -d ":" -f 2
-  portainer:
-    restart: "no"
-    image: portainer/portainer:1.15.5
-    #image: portainer/portainer:1.19.2
-    command: -H unix:///var/run/docker.sock --admin-password $PW_PORTAINER --logo $AEGEE_LOGO_B64
+
+  portal:
+    build:
+      context: ./$PATH_OMS_GLOBAL/portal/
+      dockerfile: ./Dockerfile.dev
+    image: aegee/portal:dev
+    restart: always
     expose:
-      - "9000"
-    volumes:
-      - /var/run/docker.sock:/var/run/docker.sock:rw
+      - 80
     labels:
-      - traefik.enable=true
-      - traefik.backend=portainer
-      - traefik.port=9000
-      - traefik.frontend.rule=Host:${SUBDOMAIN_PORTAINER}${BASE_URL}
-      - traefik.frontend.priority=20
+      - "traefik.enable=true"
+      - "traefik.backend=portal"
+      - "traefik.port=80"
+      - "traefik.frontend.rule=Host:${SUBDOMAIN_PORTAL}${BASE_URL}"
+      - "traefik.frontend.priority=20"
 
-  # drone-server:
-  #   image: drone/drone:0.7
-  #   ports:
-  #     - 8000
-  #   volumes:
-  #     - /var/lib/drone:/var/lib/drone/
-  #   restart: always
-  #   environment:
-  #     - DRONE_OPEN=true
-  #     - DRONE_HOST=appserver/drone
-  #     - DRONE_GITHUB=true
-  #     - DRONE_GITHUB_CLIENT=2205108af6d3aa0b1e3c
-  #     - DRONE_GITHUB_SECRET=0e84e258dd54d8b8d31fb889c01d5a7047f8db21
-  #     - DRONE_SECRET=HighWayMANN
-  #   labels:
-  #     - traefik.enable=true
-  #     - traefik.backend=drone-server
-  #     - traefik.port=8000
-  #     - traefik.frontend.rule=Host:${SUBDOMAIN_DRONE}${BASE_URL}
-  #     - traefik.frontend.priority=10
-  #
-  # drone-agent:
-  #   image: drone/drone:0.7
-  #   command: agent
-  #   restart: always
-  #   depends_on:
-  #     - drone-server
-  #   volumes:
-  #     - /var/run/docker.sock:/var/run/docker.sock
-  #   environment:
-  #     - DRONE_SERVER=ws://drone-server:8000/ws/broker
-  #     - DRONE_SECRET=HighWayMANN
-  #   labels:
-  #     - traefik.enable=false
 
 secrets:
     jwt_key:
         file: secrets/jwt_key
     sendgrid_key:
         file: secrets/sendgrid_key
     mail_user:
-        file: secrets/sendgrid_key
+        file: secrets/mail_user
     mail_password:
-        file: secrets/sendgrid_key
+        file: secrets/mail_password
     db_password:
-        file: secrets/db_password
+        file: secrets/core-db_password
```

Comment:
actually a proposal: let's delete it entirely and use env vars for it? maybe not in this PR but later

Reply:
and also for mail_user and mail_password, seems reasonable to use secret files for things where we really need files

Reply:
imho for our settings, secrets and env variables are the same thing. If an attacker gets access to a container, reading a fil in /var/run/secrets (iirc that's the folder where secrets are mounted) is not much different than reading the environment variables. I am no security expert tho, so we'll have to read about stuff.

In favour about removing secrets altogether (it'd make us able to use composer version 2.4 and `limit` resources)

Reply:
my concern is not the security here, but that'd be nice to standardize things here.
reasoning - for now we have some strings that are in .env and some strings that are in files, so this can be confusing. that's why I'd rather use .env for all of the strings and leave files only for, like, real files.

for using 2.4, why not moving to 3.x and use docker stack instead?

Reply:
i told you about the docker stack, it's a complication we don't need